### PR TITLE
Fix ESLint errors

### PR DESCRIPTION
**Pull request by mingaldrichgan**
This repo has an `.eslintrc.js` file but does not pass linting. Most of the ESLint errors were fixable with `eslint --fix`, and the rest I fixed manually.

### CODE REVIEW

File: src/Home/Project.js [hunk 1]
```diff
@@ -40,20 +39,20 @@ const OpenSourceStat = styled(({
...
       <OpenSourceStatIcon>
-        <Icon/>
+        <Icon />
       </OpenSourceStatIcon>
-      {disabled ? <div>N/A</div> :
+      {disabled ? <div>N/A</div> : (
         <div>
           <strong>{value}</strong>
-          {dataAgeInDays < 1 ? null :
+          {dataAgeInDays >= 1 &&
```
Comment (mingaldrichgan) [inline src/Home/Project.js]: Should this be `dataAgeInDays > 0` instead? I didn't want to introduce any logic changes in this PR unless the maintainers ask for it.

File: src/Home/Project.js [hunk 2]
```diff
@@ -40,20 +39,20 @@ const OpenSourceStat = styled(({
...
             <OpenSourceStatChange
               title={`${label} in the last ${dataAgeInDays} days`}
```
Comment (mingaldrichgan) [inline src/Home/Project.js]: **TODO:** Should be `day` instead of `days` when `dataAgeInDays === 1`.

File: src/Home/Project.js (hunk 1 above)
Comment (erquhart) [reply src/Home/Project.js]: `>= 1 &&` and `> 0 &&` are both equivalent to `< 1 ? null :`, so this is fine.

File: src/Home/Project.js (hunk 2 above)
Comment (erquhart) [reply src/Home/Project.js]: Yeah just didn't bother with it in the initial implementation, but the improvement is more than welcome. Makes sense to do in a separate PR.

File: src/Home/Project.js (hunk 1 above)
Comment (mingaldrichgan) [reply src/Home/Project.js]: Cool, didn't know if `dataAgeInDays` was guaranteed to be an integer.

File: src/Home/Project.js (hunk 2 above)
Comment (mingaldrichgan) [reply src/Home/Project.js]: https://github.com/netlify/headlesscms.org/pull/116

**Pull request merged**